src/js/vic-v2/components/VeteranIDCard.jsx

```diff
@@ -1,7 +1,106 @@
 import React from 'react';
 
-export default function VeteranIDCard() {
-  return (
-    <div style={{ paddingTop: '95px', backgroundColor: 'gray', height: '200px', width: '300px' }}></div>
-  );
+const FONT_FAMILY = 'Arial, Helvetica, sans-serif';
+// sets the aspect ratio of svg
+const VIEW_BOX = '0 0 2048 1284';
+// height of font top to bottom bounding box in the above viewbox
+const LARGE_FONT_SIZE = 90;
+const SMALL_FONT_SIZE = 60;
+
+const imagePaths = {
+  'Coast Guard': '/img/vic-cg-emblem.png',
+  'Air Force': '/img/vic-air-force-coat-of-arms.png',
+  Army: '/img/vic-army-symbol.png',
+  Navy: '/img/vic-navy-emblem.png',
+  'Marine Corps': '/img/vic-usmc-emblem.png',
+  previewBackground: '/img/vic-preview-background.svg',
+  VASeal: '/img/vic-va-seal.png'
+};
+
+function getIsLongName(name) {
+  try {
+    return document
+      .createElement('canvas')
+      .getContext('2d')
+      .measureText(name)
+    // manually determined max width
+      .width > 120;
```

Comment:
Didn't find a way to do this in the context of the svg's coordinate system so I tested a few widths and got this number. Will test in a few browsers to determine if the value is consistent.

Below represents the max width 1 line name. 

<img width="596" alt="screen shot 2018-02-12 at 20 12 27" src="https://user-images.githubusercontent.com/4998130/36131249-744cb8b0-1036-11e8-8835-2063ed7f86f1.png">
<img width="584" alt="screen shot 2018-02-12 at 20 13 07" src="https://user-images.githubusercontent.com/4998130/36131253-786921ae-1036-11e8-85a6-d57303d06af3.png">